cvpy/tests/test_imagetable.py

```diff
@@ -266,6 +266,139 @@ def test_imagetable_from_table_custom_col_names(self):
 
         assert image_table.table.name == cas_table.name
         assert type(image_table) == NaturalImageTable
+    
+    # Load client images and server images using load_client_images and loadImages and compare results
+    def test_imagetable_load_client_images(self):
+        # For determining if we are working from a local machine or the Jenkins server
+        # and comparing images for verification
+        from PIL import ImageChops, ImageStat
+        from os.path import exists
+
+        # Path to the directory the function will load images from
+        path = "images/"
```

Comment:
This should be the complete path to the images directory. 

path = f"{TestImageTable.DATAPATH}/data/images"